### PR TITLE
French translation for installer (.nsh)

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [x] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

### CODE REVIEW

File: installer/languages/French.nsh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+LangString MsgAlreadyInstalled ${LANG_FRENCH} '${PRODUCT_NAME} est déjà installé. $\n$\nCliquez sur « OK » pour désinstaller la version précédente ou « Annuler » pour annuler cette mise à jour.'
+LangString MsgApplicationRunning ${LANG_FRENCH} "L’application ${PRODUCT_NAME} est en cours d’exécution. Merci de la stopper et d’essayer à nouveau."
```
Comment (trebmuh) [inline installer/languages/French.nsh]: Plutôt "Veuillez" que "Merci de".

Comment (zas) [reply installer/languages/French.nsh]: Fixed in cd295cf

**Comment by phw** [comment]
@zas please uncomment French in https://github.com/metabrainz/picard/blob/master/installer/picard-setup.nsi.in#L95

I'll create some screenshots for review later

**Comment by zas** [comment]
> @zas please uncomment French in https://github.com/metabrainz/picard/blob/master/installer/picard-setup.nsi.in#L95
> 

Done in 2aafa0f

> I'll create some screenshots for review later

Ok

File: installer/languages/French.nsh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+LangString MsgAlreadyInstalled ${LANG_FRENCH} '${PRODUCT_NAME} est déjà installé. $\n$\nCliquez sur « OK » pour désinstaller la version précédente ou « Annuler » pour annuler cette mise à jour.'
+LangString MsgApplicationRunning ${LANG_FRENCH} "L’application ${PRODUCT_NAME} est en cours d’exécution. Veuillez la stopper et d’essayer à nouveau."
```
Comment (trebmuh) [inline installer/languages/French.nsh]: "Veuillez la stopper et essayer à nouveau." plutôt que "Veuillez la stopper et d’essayer à nouveau."

**Comment by phw** [comment]
So for me without speaking French this looks to be fine :D See below, I screenshotted the complete install and uninstall experience.

Keep in mind that other then the strings in our translation files we cannot affect the translation, the rest is provided by NSIS. So if something is odd or inconsistent we need to adapt our translation to their style.

grafik

grafik

grafik

grafik

grafik

grafik

grafik

grafik

grafik

grafik

grafik

File: installer/languages/French.nsh (hunk 2 above)
Comment (zas) [reply installer/languages/French.nsh]: Ooops, corrigé dans 4b2bfc2b5a0863a506192d63f4f415d6352eccc5

**Comment by zas** [comment]
> So for me without speaking French this looks to be fine :D See below, I screenshotted the complete install and uninstall experience.
> 

Thanks, I just fixed 2 small things, one being an error from me after last change, the other depends on context and your screenshots helped.

**Pull request merged**